cmake/HiOpConfig.cmake.in

```diff
@@ -0,0 +1,36 @@
+# Author(s):
+# - Cameron Rutherford <cameron.rutherford@pnnl.gov>
+
+include( "${CMAKE_CURRENT_LIST_DIR}/HiOpTargets.cmake" )
+
+# TODO: add tpl configuration for all targets that we are linking against
+if(@HIOP_USE_METIS@ AND NOT TARGET METIS)
+  add_library(METIS INTERFACE IMPORTED GLOBAL)
+  target_link_libraries(METIS INTERFACE "@METIS_LIBRARY@")
+  set_target_properties(METIS PROPERTIES EXPORT_LINK_INTERFACE_LIBRARIES "@METIS_INCLUDE_DIR@")
+endif()
+
+if(@HIOP_USE_GPU@ AND NOT TARGET hiop_cuda)
```

Comment:
No, because GPU and CUDA are still synonyms in HiOp's CMake. Subdividing GPU settings into CUDA, HIP, etc. is out of scope of this PR. This issue is tracked in #160 and solutions are prototyped in PR #198.